h/templates/embed.js

```diff
@@ -46,6 +46,17 @@
       resources = resources.concat(['{{ layout.xpath_polyfill_urls | map("string") | join("', '") | safe }}']);
     }
 
+    // https://github.com/Modernizr/Modernizr/blob/master/feature-detects/url/parser.js
+    var url;
+    try {
+      // have to actually try use it, because Safari defines a dud constructor
+      url = new URL('http://modernizr.com/');
+      var works = url.href === 'http://modernizr.com/';
```

Comment:
Don't need to. According to https://github.com/Modernizr/Modernizr/blob/master/feature-detects/url/parser.js it will throw an exception if anything is amiss. (Should I remove the works variable?)